lib/vbms/requests/create_contentions.rb

```
@@ -50,7 +50,7 @@ def soap_doc
 
                 fileNumber: @veteran_file_number,
                 claimId: @claim_id,
-                title: contention[:description],
+                title: contention[:description].slice(0, 255),
```

silently truncating text like this may surprise consumers of this library.

Could we instead test for the length and raise an exception if it is longer than 255 bytes?